src/classes/RD2_VisualizeScheduleController.cls

```diff
@@ -61,56 +61,50 @@ public with sharing class RD2_VisualizeScheduleController {
     }
 
     /**
-     * @description Retrieves the current schedule of future donations to render in the UI
-     * @param recordId
-     * @param displayNum
-     * @return List<Schedule>
+     * @description Builds future installments for the Recurring Donation
+     * @param recordId Record Id
+     * @param displayNum Number of installments to display
+     * @return List<Installment>
      */
     @AuraEnabled
     public static List<Installment> getInstallments(Id recordId, Integer displayNum) {
```

Comment:
@rponti-sforg , active schedule and projected installment components need retesting to ensure nothing is broken and behaviour is unchanged by the recent code changes.